.travis.yml

```diff
@@ -6,7 +6,6 @@ install:
   - npm install
 
 before_script:
-  - npm run installWeb3
```

Comment:
This may break the unit tests.

Reply:
Well, this will work because `npm test` runs the e2e tests first, and the `e2e-prepare-start` script installs web3. This makes the tests order-dependent, but it's not a big deal.

Reply:
@fvictorio you are right. It is better to leave in .travis.yml. I'll redefine `e2e-prepare-start` script instead.